src/Widgets/GlobalConsoleWidget.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Widgets;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_CLASS)]
```

Comment:
These globals should probably never be used for anything other than --help, --verbosity --silent etc.